boards/pyboard/include/board.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2019 Inria
+ *               2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_pyboard
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the pyboard board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
```

Comment:
Thanks, but you can remove me (and Inria) from the list of authors and copyrights.

Reply:
(same comment as above)